meson.build

```diff
@@ -226,17 +226,17 @@ if get_option('b_sanitize') == 'address'
   cdata.set('HAVE_ASAN', 1)
 endif
 
-glib_dep = dependency('glib-2.0', version: '>=2.56')
-gobject_dep = dependency('gobject-2.0')
-gmodule_dep = dependency('gmodule-2.0')
-gio_dep = dependency('gio-2.0')
-gee_dep = dependency('gee-0.8')
-json_glib_dep = dependency('json-glib-1.0')
-libsoup_dep = dependency('libsoup-2.4')
+glib_dep = dependency('glib-2.0', version: '>=2.56', allow_fallback: true)
+gobject_dep = dependency('gobject-2.0', allow_fallback: true)
+gmodule_dep = dependency('gmodule-2.0', allow_fallback: true)
+gio_dep = dependency('gio-2.0', allow_fallback: true)
+gee_dep = dependency('gee-0.8', allow_fallback: true)
+json_glib_dep = dependency('json-glib-1.0', allow_fallback: true)
+libsoup_dep = dependency('libsoup-2.4', allow_fallback: true)
```

Comment:
allow_fallback, if not specified, will default to "true if the dependency is required", and by default dependencies are required. So you do not need these kwarg changes.